pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -96,8 +98,10 @@ func AddToManager(mgr manager.Manager, ns string, rc injectingResolverClient, pr
 		CreateFunc: func(e event.CreateEvent) bool {
 			return e.Object.GetNamespace() == ns
 		},
-		DeleteFunc:  func(event.DeleteEvent) bool { return false },
-		UpdateFunc:  func(event.UpdateEvent) bool { return false },
+		DeleteFunc: func(event.DeleteEvent) bool { return false },
```

Comment:
Who will add the deletationTime stamp? I guess you need to include those changes in this PR as well, right?

Reply:
Good point. It is actually set by kubernetes when one requests a deletion, as an example: `kubectl delete pj/foobar`.
See [here](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/object-meta/).

Reply:
Aborting feature is [there](https://github.com/openshift/ci-tools/blob/9f82e2ac5526d7901623b6167fa97991f4b7b4eb/cmd/payload-testing-prow-plugin/server.go#L91-L96), a user can already abort the jobs. I think that @DennisPeriquet was also concerned about `prpqr` cleanup in his comment on https://issues.redhat.com/browse/DPTP-3819. That's the reason of this PR.